### ISSUE TITLE
An unknown backend error occurred. Please consider filing a bug!

### ISSUE THREAD

**Issue by CouchersBot** [Couchers-org/couchers#3984]
Subject: An unknown backend error occurred. Please consider filing a bug!
Description:
An unknown backend error occurred. Please consider filing a bug!

Results:
cant register

Backend version: develop-410b4008
Frontend version: develop-a38d3ac0
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36 Edg/122.0.0.0
Page: https://couchers.org/signup
User: <not logged in>

**Comment by aapeliv** [comment]
Leap day bug!!

```
ValueError: day is out of range for month
  File "couchers/interceptors.py", line 236, in tracing_function
    res = prev_func(request, context)
  File "couchers/interceptors.py", line 152, in user_unaware_function
    return user_aware_function(req, context)
  File "couchers/servicers/auth.py", line 228, in SignupFlow
    if not birthdate or birthdate >= minimum_allowed_birthdate():
  File "couchers/utils.py", line 96, in minimum_allowed_birthdate
    return today_.replace(today_.year - 18)ValueError: day is out of range for month
  File "couchers/interceptors.py", line 236, in tracing_function
    res = prev_func(request, context)
  File "couchers/interceptors.py", line 152, in user_unaware_function
    return user_aware_function(req, context)
  File "couchers/servicers/auth.py", line 228, in SignupFlow
    if not birthdate or birthdate >= minimum_allowed_birthdate():
  File "couchers/utils.py", line 96, in minimum_allowed_birthdate
    return today_.replace(today_.year - 18)
```

**Comment by aapeliv** [comment]
Fixed in b8020149f606d46a374f5565711d925329ef1ccd
